### ISSUE TITLE
Notify not available on wasm

### ISSUE THREAD

**Issue by DruggedBunny** [blitz-research/monkey2#460]
Just tried to build on wasm and found Notify isn't available, not sure if intentional...

```
Mx2cc version 1.1.15

***** Making app 'D:/Documents/Development/Sources/Monkey2 Sources/speccy island/speccy_island.monkey2' (emscripten release llvm gcc) *****

Parsing...
Semanting...
D:/Documents/Development/Sources/Monkey2 Sources/speccy island/retrofx.monkey2 [38] : Error : Identifier 'Notify' Not found

```

**Comment by engor** [comment]
This function is under `std.requesters` namespace, if you jump to definition you'll see that only `OpenUrl()` is available for Emscripten target.
So it's not a bug, but maybe there should be some replacement.

**Comment by DruggedBunny** [comment]
Yeah, probably not a bug, just unimplemented; I hacked in a __TARGET__ check to work around it.

(On the topic of wasm, I've just noticed that although all assets get bundled into a _project_.data file, it still creates an empty assets folder that seems not to be necessary.)

**Comment by seyhajin** [comment]
'Notify' is only for Desktop target, use 'OpenURL' for emscripten.
Issue can be closed.

**Comment by nitrologic** [comment]
Is window.alert() not available for this purpose?

**Comment by seyhajin** [comment]
Apparently this causes popup warnings from the browser, see 6fcdfbb.
